stable/postgresql/README.md

```diff
@@ -87,7 +88,58 @@ $ helm install --name my-release -f values.yaml stable/postgresql
 
 The [postgres](https://github.com/docker-library/postgres) image stores the PostgreSQL data and configurations at the `/var/lib/postgresql/data/pgdata` path of the container.
 
-The chart mounts a [Persistent Volume](http://kubernetes.io/docs/user-guide/persistent-volumes/) volume at this location. The volume is created using dynamic volume provisioning.
+The chart mounts a [Persistent Volume](http://kubernetes.io/docs/user-guide/persistent-volumes/) volume at this location. The volume is created using dynamic volume provisioning. Clusters configured with NFS mounts require manually managed volumes and claims.
```

Comment:
This documentation and example are for a very specific use case. I'm not sure but my feeling is that creation of physical volumes is outside the scope of this README. Im my mind this section should be shorter and more generic. The main purpose of the `existingClaim` option is to use an already exiting PVC and don't have helm create an new claim, this is generally to migrate exiting deployments to helm while keeping the data.